restlosapi.py

```diff
@@ -83,8 +91,33 @@ def __init__(self):
     def get_unique_key(self, endpoint):
         return self.endpoint_keys[endpoint]
 
+    @staticmethod
+    def _validate_timeperiod(h1, m1, h2, m2):
+        def validate_hhmm(hh, mm):
+            return (0 <= hh <= 24) and (0 <= mm <= 60) and (hh < 24 or mm == 0)
+
+        return (  # let's run a couple of rules to determine that hh:mm <= hh:mm
+            validate_hhmm(h1, m1) and
+            validate_hhmm(h2, m2) and
+            (h1 < h2 or (h1 == h2 and m1 <= m2))
+        )
+
     def validate(self, endpoint, data={}):
         for attr in data.keys():
+            if endpoint == 'timeperiod':
```

Comment:
actual solution based on suggestion: _use an if statement evereywhere to treat Timeperoids special_ from [pynag issue 243](https://github.com/pynag/pynag/issues/243)